target/arm/tlb_helper.c

```diff
@@ -193,6 +193,9 @@ bool arm_cpu_tlb_fill(CPUState *cs, vaddr address, int size,
             arm_tlb_mte_tagged(&attrs) = true;
         }
 
+#ifdef TARGET_CHERI
+        attrs.tag_setting = access_type == MMU_DATA_CAP_STORE;
+#endif
```

Comment:
These changes don't belong in this PR

Reply:
Even though the rest of the Morello support is missing here, I think keeping it as part of this commit makes more sense logically (since we do the same thing for the other architectures).